crates/edr_solidity/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "edr_solidity"
+version = "0.2.0-dev"
+edition = "2021"
+
+[dependencies]
+revm = { git = "https://github.com/Wodann/revm", rev = "93d8ebe", version = "3.3", default-features = false, features = ["dev", "serde", "std"] }
+revm-primitives = { git = "https://github.com/Wodann/revm", rev = "93d8ebe", version = "1.1", default-features = false }
```

Comment:
Done in 59776f0a650061decb6262e3ca072a78493df3b7, but I have two concerns:

1. I only did it for `revm_primitives`, not for `revm`, because `edr_eth` doesn't use that crate.
2. I had to re-export `hex_literal` from `edr_eth` too, which feels weird because the current re-exports are just types. My impression is that this is fine, but I might be wrong.